readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://xhuhm.github.io/layout_stars/)
+- [TEST REPORT LINK](https://xhuhm.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR descritpion
<img width="944" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/1cfd8482-6ae0-489e-b682-87c5374c2a47">